<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>725</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>725</width>
    <height>465</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>725</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Motion Modeling</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="xtWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>235</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="ytWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>235</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ulValueLineEdit">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>30</y>
      <width>310</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="urValueLineEdit">
    <property name="geometry">
     <rect>
      <x>407</x>
      <y>30</y>
      <width>310</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ulTimeLineEdit">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>5</y>
      <width>310</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="urTimeLineEdit">
    <property name="geometry">
     <rect>
      <x>407</x>
      <y>5</y>
      <width>310</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="xyGraphicsView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>466</width>
      <height>366</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>468</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="tlLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ulLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ul:</string>
    </property>
   </widget>
   <widget class="QLabel" name="trLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="urLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ur:</string>
    </property>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New..</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save..</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="newLeftAction">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="openLeftAction">
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
  <action name="saveLeftAction">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="saveAsAction_2">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="saveasLeftAction">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="newRightAction">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="openRightAction">
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
  <action name="saveRightAction">
   <property name="text">
    <string>Save..</string>
   </property>
  </action>
  <action name="saveasRightAction">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
